src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -211,6 +219,8 @@ fi
 			echo "'collectstatic' exited with exit code $EXIT_CODE."
 			ELAPSED_TIME=$(($SECONDS - $START_TIME))
 			echo "Done in $ELAPSED_TIME sec(s)."
+		else
```

Comment:
else indentation

Reply:
`else` seems to align correctly with `fi` on line 224

Reply:
looks like there are some mix of tabs and spaces whichc is causing this indentation .. can you convert tht tabs to spaces?

Reply:
@arroyc no spaces at the beginning of each line are displayed on my end. I'll DM you a screenshot